docs/integrations/aws-s3/readtree.md

```diff
@@ -0,0 +1,28 @@
+---
+id: read-tree
```

Comment:
Good question. I'm not entirely sure the AWS SDK supports an elegant or easy way to handle a wildcard pattern in their URLs so it may make it difficult to implement `search` in `AwsS3UrlReader`. For our use case, having `AwsS3DiscoveryProcessor` makes sense as we want to ingest our catalog entities through the `locations` section in `app-config.yaml` 

Of course, if having this custom processor does not make sense for backstage to have in the more general sense then we do not have to include it in this PR.